sdk/translation/azure-ai-translation-document/tests/test_list_submitted_jobs.py

```diff
@@ -77,4 +83,179 @@ def test_list_submitted_jobs_with_skip(self, client):
             if job.id in job_ids:
                 self._validate_translation_job(job, status="Succeeded", total=TOTAL_DOC_COUNT_IN_JOB, succeeded=TOTAL_DOC_COUNT_IN_JOB)
             else:
-                self._validate_translation_job(job)
\ No newline at end of file
+                self._validate_translation_job(job)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_status(self, client):
+        # create some jobs
+        self._create_and_submit_sample_translation_jobs(client, 10, wait=False)
+
+        # list jobs
+        statuses = ["Running"]
+        submitted_jobs = list(client.list_submitted_jobs(statuses=statuses))
+        self.assertIsNotNone(submitted_jobs)
+
+        # check statuses
+        for job in submitted_jobs:
+            self.assertIn(job.status, statuses)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_ids(self, client):
+        # create some jobs
+        job_ids = self._create_and_submit_sample_translation_jobs(client, 3)
+
+        # list jobs
+        submitted_jobs = list(client.list_submitted_jobs(ids=job_ids))
+        self.assertIsNotNone(submitted_jobs)
+
+        # check statuses
+        for job in submitted_jobs:
+            self.assertIn(job.id, job_ids)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_created_start(self, client):
+        # create some jobs
+        start = datetime.now()
+        self._create_and_submit_sample_translation_jobs(client, 3)
+
+        # list jobs
+        submitted_jobs = list(client.list_submitted_jobs(created_date_time_utc_start=start))
+        self.assertIsNotNone(submitted_jobs)
+
+        # check statuses
+        for job in submitted_jobs:
+            self.assert(job.created_on > start)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_created_end(self, client):
+        # create some jobs
+        end = datetime.now()
```

Comment:
should we be grabbing this timestamp after creating the translation jobs?

Reply:
good point!
i was thinking of grapping old jobs
but now i'm thinking this might cause a problem 
so i need to create two jobs